TimeManagement.py

```diff
@@ -0,0 +1,44 @@
+"""
+Time Management portion of the project
+"""
+from datetime import datetime
+from datetime import timedelta
+from datetime import datetime
+import os
+from playsound import playsound
+import time
+
+class TimeManagment():
+
+    def __init__(self):
+        self.time_start = datetime.now()
+        self.work_duration_hours = int(input("How many hours would you like to work?\n"))
+        self.work_duration_minutes = int(input("How many minutes would you like to work?\n"))

```

Comment:
call the functions you created within the constructor... I think we want this to start write when we activate the script